src/__tests__/contracts.ts

```diff
@@ -22,11 +22,12 @@ import { Contract, Version, Project, Network } from '../entity/Contract';
 import { EncryptorContainer } from '../entity/encryption/EncryptorContainer';
 import { EthereumService } from "../eth/EthereumService";
 
+import { latest as LATEST_NOTARY } from '../../rpc/VaultNotary/Latest';
```

Comment:
For consistency, let's define this one near line 30 and hard code the version `1.0.0` like the other LATEST_* constants.  Since we are explicitly testing specific versions of the contracts, we don't actually want this to import the latest version like we do elsewhere.